front/tests/009-stdcm-between-two-trains.spec.ts

```diff
@@ -0,0 +1,153 @@
+import { test, expect } from '@playwright/test';
+import { PlaywrightHomePage } from './pages/home-page-model';
+import PlaywrightRollingstockModalPage from './pages/rollingstock-modal-model';
+import { PlaywrightSTDCMPage } from './pages/stdcm-page-model';
+import PlaywrightMap from './pages/map-model';
+import VARIABLES from './assets/operationStudies/testVariables';
+import PATH_VARIABLES from './assets/operationStudies/testVariablesPaths';
+import PlaywrightScenarioPage from './pages/scenario-page-model';
+import { ProjectPage } from './pages/project-page-model';
+import { StudyPage } from './pages/study-page-model';
+
+const pathfindingDistance = VARIABLES.stdcm.pathfindingDistance
```

Comment:
This is enough, you just need to test that we have 
```suggestion
const pathfindingDistance = VARIABLES.stdcm
```

Reply:
ty, fixed

Reply:
done